src/tests/test_api.py

```diff
@@ -0,0 +1,171 @@
+import unittest
+import xmlrpclib
+import base64
+import json
+from time import sleep
+
```

Comment:
Remove empty line

Reply:
pep8 says the opposite: You should put a blank line between each group of imports.
https://www.python.org/dev/peps/pep-0008/#imports